src/view/element.js

```diff
@@ -712,6 +712,48 @@ export default class Element extends Node {
 			( attributes == '' ? '' : ` ${ attributes }` );
 	}
 
+	/**
+	 * Creates element instance from provided viewElementDefinition.
+	 *
+	 * @param {module:engine/view/viewelementdefinition~ViewElementDefinition} viewElementDefinition
+	 * @returns {Element}
+	 */
+	static fromViewDefinition( viewElementDefinition ) {
```

Comment:
Misses a verb and repeating "view" isn't necessary IMO. `ViewElement.createFromDefinition()`. should be fine.

Reply:
Note that soon view writer will be the only proper way of changing model, so such methods will have to be moved.